src/main/java/net/sf/mzmine/modules/visualization/peaklisttable/PeakListTablePopupMenu.java

```diff
@@ -166,7 +168,8 @@ public PeakListTablePopupMenu(final PeakListTableWindow window, PeakListTable li
 
     searchMenu = new JMenu("Search");
     add(searchMenu);
-    dbSearchItem = GUIUtils.addMenuItem(searchMenu, "Search online database", this);
+    onlineDbSearchItem = GUIUtils.addMenuItem(searchMenu, "Search online database", this);
+    spectralDbSearchItem = GUIUtils.addMenuItem(searchMenu, "Search spectral database", this);
```

Comment:
Let's change these two menu items to "Search online compound database" and "Search local spectral database"

Reply:
okay